Documentation/prereqs.md

```diff
@@ -0,0 +1,51 @@
+.NET Core native prerequisites
+==============================
+
+This document outlines the dependencies needed to run .NET Core tools and applications. 
+
+## Windows dependencies
+On Windows, the only dependency is the VC++ Redistributable. Depending on the version of Windows you are running on, the versions are changing.
```

Comment:
Also, see:

https://social.technet.microsoft.com/Forums/en-US/e08daebc-c298-4932-a9d7-ff08fabf9536/nano-server-packages?forum=NanoServer

Some formal definition at https://blogs.technet.microsoft.com/windowsserver/2015/11/16/moving-to-nano-server-the-new-deployment-option-in-windows-server-2016/